WcaOnRails/app/controllers/server_status_controller.rb

```diff
@@ -2,6 +2,22 @@
 class ServerStatusController < ApplicationController
   MINUTES_IN_WHICH_A_JOB_SHOULD_HAVE_STARTED_RUNNING = 5
 
+  def self.bad_keys_by_type_by_locale
+    @bad_keys_by_type_by_locale ||= begin
+      bad_keys_by_type_by_locale = {}
```

Comment:
Oh, Jonatan remembered that this is static data, so we can make it a constant!!

Yes, of course!

It should be `BAD_I18N_KEYS` in that case.